mock/mock.spec

```diff
@@ -257,6 +262,7 @@ pylint-3 py/mockbuild/ py/*.py py/mockbuild/plugins/* || :
 %config(noreplace) %{_sysconfdir}/%{name}/*.ini
 %config(noreplace) %{_sysconfdir}/pam.d/%{name}
 %config(noreplace) %{_sysconfdir}/security/console.apps/%{name}
+%{_sysusersdir}/mock.conf
```

Comment:
Hmm, shouldn't we keep this in mock-filesystem?

Reply:
Shall this be `%config`?

Reply:
Still valid.